src/content/governance/index.md

```diff
@@ -0,0 +1,151 @@
+
+<ButtonLink to="/eips/">More on EIPs</ButtonLink>
+
+<Divider />
+
+## The Formal Process {#formal-process}
+
+The formal process for introducing changes to the Ethereum protocol is as follows:
+
+1. **Propose a Core EIP**: as described in [EIP-1](https://eips.ethereum.org/EIPS/eip-1#core-eips), the first step to formally proposing a change to Ethereum is to detail it in a Core EIP. This will act as the official specification for an EIP that Protocol Developers will implement if accepted.
+
+2. **Present your EIP to Protocol Developers**: once you have a Core EIP, you should get feedback on it from Protocol Developers. You can get feedback can by proposing it for discussion on an [AllCoreDevs call](https://github.com/ethereum/execution-specs/tree/master/network-upgrades#getting-the-considered-for-inclusion-cfi-status). Alternatively, discussion can happen asynchronously on the [Ethereum Magician's forum](https://ethereum-magicians.org/) or in the [Ethereum R&D Discord](https://discord.gg/mncqtgVSVw).
+
+> Potential outcomes of this stage are:
+
+> - The EIP will be considered for a future network upgrade
+> - Technical changes will be requested
+> - It may be rejected if it is not a priority or the improvement is not large enough relative to the development effort.
+
+3. **[Optional] Gather Community Consensus for your EIP**: some EIPs are purely technical changes with little room for debate, but others can be more contentious. If the EIP is in the latter category, Protocol Developers will likely want to see strong evidence of overwhelming community support before proceeding with it. There is no "one size fits all" approach here, as each EIP is unique.
+
+4. **EIP Included in Network Upgrade**: assuming the EIP is approved, tested and implemented, it gets scheduled as part of a network upgrade. Given the high coordination costs of network upgrades (everyone needs to upgrade simultaneously), EIPs are generally bundled together in upgrades.
+
+5. **Network Upgrade Activated**: after the network upgrade is activated, the EIP will be live on the Ethereum network. _Note: network upgrades are usually activated on testnets before being activated on the Ethereum Mainnet._
+
+This flow, while very simplified, gives an overview of the significant stages for a protocol change to be activated on Ethereum. Now, let's look at the informal factors at play during this process.
+
+## Informal Process {#informal-process}
+
+### Understanding Prior Work {#prior-work}
+
+EIP Champions should familiarise themselves with prior work and proposals before having an EIP, which can be seriously considered for deployment on the Ethereum Mainnet. This way, the EIP hopefully brings something new which hasn't been rejected before. The three main places to research this are the [EIP repository](https://github.com/ethereum/eips), [Ethereum Magicians](https://www.ethereum-magicians.org/) and [ethresear.ch](https://www.ethresear.ch/).
```

Comment:
"before having an EIP"? Not sure I follow what that means. Before creating an EIP?